UI/setup/edit_user.html

```diff
@@ -50,9 +50,12 @@
                 <td><label for="password">[% text('Password') %]</label>
                 </td>
                 <td>
-                    <input type="password"
-                           id="password" name="password" value=""
-                           autocomplete="off" />
+                  [% PROCESS input element_data = {
```

Comment:
Hmm. why did you include this commit in the PR? It's already on master, no?

Reply:
Rebase to remove it